shuup/admin/template_helpers/shuup_admin.py

```diff
@@ -44,6 +44,17 @@ def get_menu_entry_categories(context):
     return menu.get_menu_entry_categories(request=context["request"])
 
 
+def is_menu_category_active(category, target_url):
+    def does_identifier_match(always_active_identifier):
+        return bool("%s" % category.identifier == "%s" % always_active_identifier)
```

Comment:
Probably the bool cast is redundant, simpler way:
```
return "%s" % category.identifier == "%s" % always_active_identifier
```

Reply:
`force_text(category.identifier) == force_text(always_active_identifier)`